### DISCUSSION TITLE
Allow Synchronization with "Computers" files of Google Drive

### DISCUSSION THREAD

**Discussion by scarbajali** [damionx7/Zettel-Notes-Documentation#143] [Feature Requests]
Google Drive stores data in two places, one is called "My Drive" and the other one is "Computers". Currently, it is only possible to synchronize with any folder from the first one, but the second one is not accessible from the app.

**Comment by damionx7** [comment]
Google drive sync will be removed in upcoming versions. You can read more about the reason here https://t.me/zettelnotes/345.

**Reply by sevenate** [reply]
What would be the advice for users, who had Google drive sync setup for the folder with previous app version and now after app auto-updated to the newer version _without_ GDrive integration - can't even edit current synchronization settings for that folder to change it to something else that is supported like GitHub?

**Reply by damionx7** [reply]
Git synchronization can be setup on newly created repositories only (as underlying library jgit only works with local folders). However, you can setup webdav or dropbox synchronization for these repositories in repository settings (shown on repository long press).
